Tests/tests/test_SystemServices.cpp

```diff
@@ -333,31 +336,61 @@ TEST_F(SystemServicesEventTest, Timezone)
                                           "\"params\":"
                                           "{"
                                           "\"oldTimeZone\":\"America\\/New_York\","
-                                          "\"newTimeZone\":\"America\\/Costa_Rica\""
+                                          "\"newTimeZone\":\"America\\/Costa_Rica\","
```

Comment:
Pls ensure that we have tests to verify that the event is fired even if one of the values change (Timezone or Accuracy)